src/widgets/refinement-list/refinement-list.js

```diff
@@ -134,10 +134,11 @@ function refinementList({
       return createURL(state.toggleRefinement(attributeName, facetValue));
     }
 
-    // Pass count of currently selected items to the header template
+    // Pass count of total and currently selected items to the header template
+    const facetsCount = facetValues.length;
```

Comment:
So do you think we should go through with changes to this PR, or is using a v2 connector the prescribed approach?